build.gradle

```diff
@@ -457,7 +540,7 @@ configure(project(':desktop')) {
     apply plugin: 'com.github.johnrengelman.shadow'
     apply from: 'package/package.gradle'
 
-    version = '0.0.1-SNAPSHOT'
+    version = '1.0.0-SNAPSHOT'
```

Comment:
We should update the version in Version.java too.

Reply:
Yes, changing.